### PR TITLE
I call this one... Bold and Brash

### PR DESCRIPTION
**Pull request by Wilchenx**
https://www.youtube.com/watch?v=Fmct2OAzE0g PR theme

## About The Pull Request

Adds a new playable squid species

image

Dedicated to squidqueen who complained about lack of them

Thanks to @WJohn for the amazing sprite!

Naming scheme:

Change the first letter of a human name
into Squ
unless the second letter is a o
then use Sqo

## Why It's Good For The Game

Something like this was never done before

Their weakness is not being able to wear shoes

## Changelog
:cl:
add: Squidpeople are a thing now
/:cl:

Todo:

Add low pressure weakness
Add high pressure resistance
Add ink spray ability
Add R'lyehian (https://www.yog-sothoth.com/wiki/index.php/R'lyehian)

### CODE REVIEW

**Comment by Wilchenx** [comment]
The species also only like canned bread as a food (future pr)

image

**Comment by swagbringer** [comment]
Are they allowed ERP with their sexy tentacles?

**Comment by Wilchenx** [comment]
Only if server rules allow for it

**Comment by habader** [comment]
this better not be a roundstart race

**Comment by Wilchenx** [comment]
Whats wrong with them being roundstart

**Comment by SouDescolado** [comment]
> this better not be a roundstart race

Yea. this should not be an option.

It should be **ENFORCED**. SquidStation13

**Comment by Wilchenx** [comment]
SquidStation13

You meant haborstation

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    race = /datum/species/squid
+
+/datum/species/squid/qualifies_for_rank(rank, list/features)
+    return TRUE
+
+/datum/species/squid/random_name(gender,unique,lastname)
+	if(unique)
+		return random_unique_squid_name()
+
+	var/randname = squid_name()
```
Comment (bobbahbrown) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm]: needs gender argument for random naming both male and female

**Comment by SouDescolado** [comment]
You should make an elder god variant where i͕̤t̤̝̻̲̺ͅ ̦̺c̪̞̟̬͍̞ͅo͕̣̥͉͍͡ͅm͕͎͖̹̹̦͇e̛̥̣̳͇̤̠̗s̫ gives damage to anyone who ṟ̴̯͍e͍̬p̸͍̼̹̰͉̞̱e̬̦n̩̱t̯ hears it speak. Also make it d̸̺̮e̘̯̳̜̦̘̮a͚̱t̤͍̭h̡̤̙̰͔̩ ̼͎̰̟̥i̥̹̱̱̳͇͓͟s̷̺̗͔̥̠̟ ̸i͘n̘͔̹̞e͓̰v̭́ḭ̛t͖̮̟̥̞͇̀a̖̯̟̝̥b͖̮̺̘̼̳̱͡l͍͟e not understand what it says and give it a really mundane objective like "get hugged by the captain" or something

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/datum/species/squid
+    name = "Squidperson"
+    id = "squid"
+    default_color = "b8dfda"
+    species_traits = list(MUTCOLORS,EYECOLOR)
```
Comment (bobbahbrown) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm]: add
`inherent_traits = list(TRAIT_RESISTHIGHPRESSURE)`
for high pressure resistance

Comment (Wilchenx) [reply code/modules/mob/living/carbon/human/species_types/squidpeople.dm]: i know of that, but doesn't that just give stupid amounts of pressure resistance?
Like full immunity?

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		return random_unique_squid_name()
+
+	var/randname = squid_name(gender)
+
+	if(lastname)
+		randname += " [lastname]"
+
+	return randname
+
+/proc/random_unique_squid_name(attempts_to_find_unique_name=10)
```
Comment (bobbahbrown) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm]: Include gender argument here

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    for(var/i in 1 to attempts_to_find_unique_name)
+        . = capitalize(squid_name())
```
Comment (bobbahbrown) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm]: Use the now included gender argument here

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    name = "Squidperson"
+    id = "squid"
+    default_color = "b8dfda"
+    species_traits = list(MUTCOLORS,EYECOLOR)
+    default_features = list("mcolor" = "FFF") // bald
+    changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | SLIME_EXTRACT
+    use_skintones = 0
+    no_equip = list(SLOT_SHOES)
+    skinned_type = /obj/item/stack/sheet/animalhide/human
+    disliked_food = FRIED
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm]: This is brilliant.

**Comment by ATH1909** [comment]
Real talk, though, I'd totally be down for adding squidpeople as a roundstart race as a non-joke.

**Comment by ExcessiveUseOfCobblestone** [comment]
lol...

**Comment by Wilchenx** [comment]
@ExcessiveUseOfCobblestone it has some april fools code in it for today only, but still see the possiblity of them becoming a normal race, maybe not roundstart

**Comment by ExcessiveUseOfCobblestone** [comment]
if this is a serious pr you need to provide proof that a maintainer has okayed this since it's the addition of a new race.

**Comment by Wilchenx** [comment]
oh i thought it was only required for roundstart races

**Comment by AffectedArc07** [comment]
good damnit shezza

**Comment by Tacolizard** [comment]
hot

**Comment by lmwevil** [comment]
hello yes i take full credit for asking for squidpeople

**Comment by squidlover** [comment]
You know what? Merge this. Actually merge this. This is the kind of silliness that fits with the game perfectly.

**Comment by Wilchenx** [comment]
image

You know where to take it to if you want to see these for real.

**Comment by ExcessiveUseOfCobblestone** [comment]
who is Thog?

**Comment by PKPenguin321** [comment]
me thog me headcoder me merge prs me close other prs ungh ungh *beats chest*

**Pull request closed without merging**